### DISCUSSION TITLE
Output Problem

### DISCUSSION THREAD

**Discussion by venil-04** [microsoft/vscode-cpptools#8048] [Q&A]
I Faced a major problem in vs code . When i Try to Run my code it shows only half output.

Example: If output was 1 2 3 4 5 6 . Then Terminal give 1 2 Output.

How can i resolve this please help!

**Comment by sean-mcmanus** [comment]
What is your code? Are you flushing the output?

**Reply by venil-04** [reply]
#include <bits/stdc++.h>
using namespace std;



int main()
{
    int n;
    int count;
    int arr[n];

    cout << "the size of array " << endl;
    cin >> n;
    cout << "elements:" << endl;
    for (int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }

    // Direct Sorting

    for (int i = 0; i < n; i++)
     {
        for (int j = i + 1; j < n; j++)
        {
            if (arr[j] < arr[i])
            {
                int temp;
                temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
     }

     for (int i = 0; i < n; i++)
    {
        cout << arr[i] << " ";
    }
return 0;
}

**Reply by sean-mcmanus** [reply]
You need to add a call to std::flush or std::endl to flush the buffer before exiting.
